src/main/java/mServer/crawler/sender/MediathekDw.java

```diff
@@ -34,6 +34,8 @@
 public class MediathekDw extends MediathekReader implements Runnable {
 
     public final static String SENDERNAME = Const.DW;
+    private final static String ADDURL = "http://tv-download.dw.de/dwtv_video/flv/";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)